README.md

```diff
@@ -69,6 +69,40 @@ Note : app.json does not contain `DEBUG` variable as it explicitly configures sc
 
 
 
+## Migration
+
+export_database.py contains script to migrate the default database
+
+### Usage
+
+Script takes two default arguments:
+
+1. URI for source database using flag
+
+    `--source, -s`
+
+2. URI for target database using flag
+
+    `--target, -t`
+
+If the script encounters any problem during the migration with any of the documents, it pushes them to defaulters.bson<br/>
+If a document was already present in the target database, it pushes them to repeated_notices.bson
+
+If you want to further try to push from a bson file, specify it with argument
+<br/>`--try-defaulters or -d`
+
+Examples:
+
+1. Default Export:
+
+    `python export_database.py --source "<Source DB URI>" --target "<Target DB URI>"`
+
+2. Adding documents from bson dump file (generated by script in case of errors) : defaulters.bson
+
+    `python export_database.py --source "<Source DB URI>" --target "<Target DB URI>" --try-defaulters defaulters.bson`
```

Comment:
Why does defaulters require source DB URI?

Reply:
Ohh yes! This might cause confusion. I'll remove the source if defaulter is supplied